examples/react-app/package.json

```diff
@@ -8,10 +8,15 @@
     "build": "vite build"
   },
   "dependencies": {
+    "@fuel-wallet/react": "0.15.2",
+    "@fuel-wallet/sdk": "0.15.2",
```

Comment:
Remove the `@fuel-wallet/sdk` and replace `@fuel-wallet/react` with `@fuels/react`